docs/customize-survey-creation-process.md

```diff
@@ -21,15 +21,16 @@ Adorners are design-surface controls that allow Survey Creator users to manipula
-| `allowChangeRequired`  | Shows or hides the adorner that makes the question required. |
-| `allowChangeType`      | Shows or hides the adorner that changes the survey element type.      |
+| Property               | Description                                                                                                                                                                                             |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `allowChangeRequired`  | Shows or hides the adorner that makes the question required.                                                                                                                                            |
+| `allowChangeType`      | Shows or hides the adorner that changes the survey element type.                                                                                                                                        |
 | `allowChangeInputType` | Shows or hides the adorner that changes the [`inputType`](https://surveyjs.io/form-library/documentation/api-reference/text-entry-question-model#inputType) property of Single-Line Input questions.    |
-| `allowCopy`    | Shows or hides the adorner that duplicates the survey element.  |
-| `allowDelete`  | Shows or hides the adorner that deletes the survey element.  |
-| `allowDragging`  | Shows or hides the adorner that allows users to drag and drop survey elements.      |
-| `allowEdit`    | Shows or hides the adorners that allow users to edit survey element properties on the design surface. If you disable this property, users can edit survey element properties only in the Property Grid. |
+| `allowCopy`            | Shows or hides the adorner that duplicates the survey element.                                                                                                                                          |
+| `allowDelete`          | Shows or hides the adorner that deletes the survey element.                                                                                                                                             |
+| `allowDragging`        | Shows or hides the adorner that allows users to drag and drop survey elements.                                                                                                                          |
+| `allowEdit`            | Shows or hides the adorners that allow users to edit survey element properties on the design surface. If you disable this property, users can edit survey element properties only in the Property Grid. |
+| `allowShowEditor`      | Shows or hides the adorner that edits the survey element                                                                                                                                                |
```

Comment:
Here is the change, most of the other places are formating, as sizes of the columns were inconsistent